### PR TITLE
Fixed description of SearchExpressions value "none"

### PR DESCRIPTION
**Pull request by ralfhandl**
Fixes https://issues.oasis-open.org/browse/ODATA-1453

### CODE REVIEW

**Review by HeikoTheissen — COMMENTED** [review]
So 0 has the same meaning as 7 (= 1 + 2 + 4)?

**Comment by ralfhandl** [comment]
> So 0 has the same meaning as 7 (= 1 + 2 + 4)?

`UnsupportedExpressions: 0` means "you can use all expressions"

`UnsupportedExpressions: 7` means "you can't use AND, OR, and NOT"

File: vocabularies/Org.OData.Capabilities.V1.xml [hunk 1]
```diff
@@ -544,7 +544,7 @@ supported:
       </ComplexType>
       <EnumType Name="SearchExpressions" IsFlags="true">
         <Member Name="none" Value="0">
-          <Annotation Term="Core.Description" String="Single search term" />
+          <Annotation Term="Core.Description" String="No restrictions, all search terms are supported" />
         </Member>
         <Member Name="AND" Value="1">
           <Annotation Term="Core.Description" String="Multiple search terms separated by `AND`" />
```
Comment (HeikoTheissen) [inline vocabularies/Org.OData.Capabilities.V1.xml:550]:
```suggestion
          <Annotation Term="Core.Description" String="Multiple search terms separated by `AND` are not supported" />
```
and likewise for values 2 and 4.

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]
Let others avoid the misunderstanding I suffered.

File: vocabularies/Org.OData.Capabilities.V1.xml [hunk 2]
```diff
@@ -543,23 +543,24 @@ supported:
...
-          <Annotation Term="Core.Description" String="Search terms preceded by `NOT`" />
+          <Annotation Term="Core.Description" String="Search terms preceded by `NOT` are not supported" />
         </Member>
         <Member Name="phrase" Value="8">
-          <Annotation Term="Core.Description" String="Search phrases enclosed in double quotes" />
+          <Annotation Term="Core.Description" String="Search phrases enclosed in double quotes are not supported" />
         </Member>
         <Member Name="group" Value="16">
-          <Annotation Term="Core.Description" String="Precedence grouping of search expressions with parentheses" />
+          <Annotation Term="Core.Description" String="Precedence grouping of search expressions with parentheses are not supported" />
```
Comment (HeikoTheissen) [inline vocabularies/Org.OData.Capabilities.V1.xml:563]:
```suggestion
          <Annotation Term="Core.Description" String="Precedence grouping of search expressions with parentheses is not supported" />
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Pull request closed without merging**